### PR TITLE
IDMP-528 - The definition of physical substance in the ISO 11238 ontology should correspond to the precise wording from paragraph 3.84

### PR DESCRIPTION
**Pull request by ElisaKendall**
Description: Revised the substances definitions of physical manufactured item and physical substance to add annotations stating that they are extensions of the ISO 11238 standard and thus their definitions do not need to conform precisely to the wording in the standard.

Signed-off-by: Elisa Kendall [ekendall@thematix.com](mailto:ekendall@thematix.com)

### CODE REVIEW

File: ISO/ISO11238-Substances.rdf [hunk 1]
```diff
@@ -1112,8 +1112,9 @@
 		</rdfs:subClassOf>
 		<rdfs:label>physical manufactured item</rdfs:label>
 		<dct:source>ISO 11238:2018 Health informatics - Identification of medicinal products (IDMP) - Data elements and structures for the unique identification and exchange of regulated information on substances, clause 3.38</dct:source>
-		<skos:definition>good that embodies the qualitative and quantitative composition of a product as contained in the packaging of the medicinal product</skos:definition>
+		<skos:definition>qualitative and quantitative composition of a product as contained in the packaging of the medicinal product</skos:definition>
```
Comment (tw-osthus) [inline ISO/ISO11238-Substances.rdf:1115]: This is the definition for the specification of the manufactured item. It is wrong or at least very misleading as definition of the physical manufactured item. Note that we have the product constituency which would better fit this definition.

**Review by tw-osthus — CHANGES_REQUESTED** [review]

**Comment by ElisaKendall** [comment]
@tw-osthus Amgen specifically requested that we change the definition for this element to the language from the specification though. So if the definition was more appropriate as we had it, we will need to explain fully to Amgen.

**Comment by tw-osthus** [comment]
Amgen needs to understand the difference between the physical manufactured item and the manufactured item. The physical manufactured item is only an interface for an IDMP specification to the inventory or a specific case or study, where the physical MI is used. Changing the definition to something, that does not match the formal definitions (class hierarchy and restrictions) is an error that must be fixed.

The PMI is a product, and it embodies a composition, but it is not a composition. That would be a dependent quality/characteristic of the independent physical thing, that is packaged innermost in the PMP, so the definition is wrong.

**Comment by ElisaKendall** [comment]
@tw-osthus After discussion with Heiner earlier today, we agreed that the original definition was fine and ensured that there is an annotation stating that physical manufactured item is specifically an extension to the standard. This way we can defend the fact that the definition isn't identical to the original.

**Review by tw-osthus — DISMISSED** [review]

**Review by tw-osthus — APPROVED** [review]

**Pull request merged**